requirements.txt

```diff
@@ -4,6 +4,8 @@ gpiozero
 picamera
 sounddevice
 sanic
+gettext
```

Comment:
no need to require gettext and locale, they are builtin with python3, they can't be installed individually